src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.KeyVault.tests.ps1

```diff
@@ -35,6 +35,65 @@ Describe "Arcus" {
                 $expected.permissions.certificates | Should -Be $certificatePermissions
                 $expected.permissions.storage | Should -Be $storagePermissions
             }
+            It "Set secret in Key Vault" {
+                $contents = "this is the raw secret certificate field contents"
+                $file = New-Item -Path "test-file.txt" -ItemType File -Value $contents
```

Comment:
To be fully correct, this should probably be in an integration test project, but since another PR already creates this project, I don't want to have too many merge conflicts, so I'll leave it here for now. Later we can move this once all the PR's are on `master`.